src/app/modules/model-version/containers/model-version-details/model-version-details.component.html

```diff
@@ -1,5 +1,13 @@
 <div class="model-version">
   <div *ngIf="modelVersion$ | async as modelVersion">
+    <div class="model-version__row" *ngIf="liftMetadata">
```

Comment:
Do not duplicate html.
If it isn't possible lift block by css, use https://angular.io/api/common/NgTemplateOutlet